src/components/searchForm/searchForm.tsx

```diff
@@ -0,0 +1,19 @@
+import './searchForm.css';
+import { useState } from 'react';
+const SearchForm = ({initialSearchQuery, onSearch}: SearchFormProps) => {
```

Comment:
Better hold the 'SearchFormProps' inside the component. 
1) It is more useful to see what are those props without walking into another folder
2)Any other component will use this declaration - there is no sense to hold it in the separate file.

The same for all props interfaces